examples/hello-triangle/README.md

```diff
@@ -0,0 +1,13 @@
+# hello-triangle
+
+This example renders a triangle to a native window.
```

Comment:
This could probably be "window" instead of "native window" because it also runs on the web now :)

Reply:
Will do! The running on the web part was what got me starting to take a look :)